wandb/sdk/launch/agent/agent.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, entity: str, project: str, queues: Iterable[str] = None):
             entity, project, queues, self.gorilla_supports_agents
         )
         self._id = create_response["launchAgentId"]
+        self._name = ""  # hacky: want to display this to the user but we don't get it back from gql until polling starts. fix later
```

Comment:
Couldn't we make the response to CreateLaunchAgent provide this? Or would that break on old versions of the CLI when updated?

Reply:
discussed, we'll leave it for now — would require an extra introspection call to avoid breaking gorilla side compatibility